libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -130,6 +130,9 @@ void AP_AHRS_NavEKF::update(bool skip_ins_update)
         // update optional alternative attitude view
         _view->update(skip_ins_update);
     }
+
+    // update NMEA output
+    update_nmea_out();
```

Comment:
is update_nmea_output() needed both here and in DCM? We always run DCM, so should be OK in one entry point

Reply:
I don't mind running it just here - we still have support for adding just DCM, that was the reason for adding both places).

Running it only in DCM would be OK-ish, but it would delay the current output from EKF (because DCM is updated before EKF is).

Reply:
I ended up removing this from DCM (and View, which was actually a bug) since @peterbarker correctly pointed out that code depends on AHRS_NavEKF being available.